cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest

```diff
@@ -12,11 +14,15 @@
 {% if grains.cloud == 'gce' -%}
   {% if grains.cloud_config is defined -%}
     {% set cloud_config = "--cloud_config=" + grains.cloud_config -%}
+    {% set cloud_config_mount = "{\"name\": \"cloudconfigmount\",\"mountPath\": \"" + grains.cloud_config + "\", \"readOnly\": true}," -%}
```

Comment:
the logic for gce and aws is the same.
can you use an 'or' clause ? http://jinja.pocoo.org/docs/dev/templates/#list-of-control-structures